index.d.ts

```diff
@@ -0,0 +1,33 @@
+declare module "react-native-get-location" {
+  export class Location {
+    latitude: number;
+    longitude: number;
+    altitude: number;
+    accuracy: number;
+    speed: any;
+    time: number;
+    bearing?: number;
+    provider?: number;
+    verticalAccuracy?: number;
+    course?: number;
+  }
+  export interface GetCurrentPositionOptions {
+    enableHighAccuracy: boolean;
+    timeout: number;
+  }
+
+  export type LocationErrorCode =
+    | "CANCELLED"
+    | "UNAVAILABLE"
+    | "TIMEOUT"
+    | "UNAUTHORIZED";
+
+  export class LocationError extends Error {
+    name: string;
```

Comment:
Here we have a `message? : string` instead of `name`.

Reply:
I had a quick look at Error and it already exposes: 
`message: string, name: string`
So I have removed the `name: string` override.